src/web/app/components/question-types/question-statistics/question-statistics.ts

```diff
@@ -41,6 +43,34 @@ export class QuestionStatistics<Q extends FeedbackQuestionDetails, R extends Fee
     this.question = question;
   }
 
+  static appendStats = (prevStats: string, newStats: string): string => {
```

Comment:
I am wondering how this still works... right now the stats calculation is done on the backend and it looks like the method relies on all data being there. Now that it is split by section/type, it feels like there is going to be some edge cases here

Reply:
At the moment, the backend will actually generate empty stats with invalid (e.g. -9999) data for the team members who have no response yet (including those whose entries are not yet populated). 

For example:
team 1 with Alice, Bob and Charlie; only Alice has responded:
Stats returned will be `{ Alice: <valid stats>, Bob: <invalid/placeholder stats>, Charlie: <invalid/placeholder stats>}`

This function is relying on that the backend continues to output in the same format. What do you think about this?

Reply:
At the moment, the backend will actually generate empty stats with invalid (e.g. -9999) data for the team members who have no response yet (including those whose entries are not yet populated).

For example:
team 1 with Alice, Bob and Charlie; only Alice has responded:
Stats returned will be { Alice: <valid stats>, Bob: <invalid/placeholder stats>, Charlie: <invalid/placeholder stats>}

This function is relying on that the backend continues to output in the same format. What do you think about this?

Reply:
Btw, the above observation was made after countless attempts trying to decipher the backend working of this part; the current logic is a bit arcane and front/backend data flow seems not very integral yet. I see TODO somewhere (about the stats struct which is currently still in a string form and requires jsonification), which might be the next goal to fix.

Reply:
Yes it conveniently works out because this is a team level contribution. The data returned by backend will be correct for that section. We just need to keep track of the new data as you have done.

Also remove lines 48-50 as they do nothing.

Reply:
Oh will fix that now...